### PR TITLE
Tags menu/muted words improvements

### PR DESCRIPTION
**Pull request by estrattonbailey**
Fixes some usability issues that cropped up

CleanShot 2024-02-27 at 15 35 57@2x
CleanShot 2024-02-27 at 15 37 03@2x
CleanShot 2024-02-27 at 15 37 11@2x
CleanShot 2024-02-27 at 15 48 57@2x

### CODE REVIEW

File: src/components/TagMenu/index.web.tsx [hunk 1]
```diff
@@ -119,7 +125,10 @@ export function TagMenu({
         accessibilityLabel={_(msg`Click here to open tag menu for ${tag}`)}
         accessibilityHint=""
         // @ts-ignore
-        items={dropdownItems}>
+        items={dropdownItems}
+        triggerStyle={web({
+          textAlign: 'left',
+        })}>
```
Comment (estrattonbailey) [inline src/components/TagMenu/index.web.tsx:131]: Mobile web, loooong tag was center aligned, maybe default button css 🤔

File: src/components/dialogs/MutedWords.tsx [hunk 2]
```diff
@@ -260,9 +260,21 @@ function MutedWordRow({
...
+            a.flex_1,
+            a.leading_snug,
+            a.w_full,
+            a.font_bold,
+            t.atoms.text_contrast_high,
+            web({
+              overflowWrap: 'break-word',
+              wordBreak: 'break-word',
+            }),
+          ]}>
```
Comment (estrattonbailey) [inline src/components/dialogs/MutedWords.tsx:277]: Support breaking to multiple lines

File: src/lib/strings/helpers.ts [hunk 3]
```diff
@@ -8,10 +8,27 @@ export function pluralize(n: number, base: string, plural?: string): string {
   return base + 's'
 }
 
-export function enforceLen(str: string, len: number, ellipsis = false): string {
+export function enforceLen(
+  str: string,
+  len: number,
+  ellipsis = false,
+  mode: 'end' | 'middle' = 'end',
+): string {
```
Comment (estrattonbailey) [inline src/lib/strings/helpers.ts:16]: Adds `mode` with backwards compat for `end` ellipsis and fallbacks

File: src/view/com/util/forms/NativeDropdown.web.tsx [hunk 4]
```diff
@@ -120,7 +122,8 @@ export function NativeDropdown({
           accessibilityLabel={accessibilityLabel}
           accessibilityHint={accessibilityHint}
           onPress={() => setOpen(o => !o)}
-          hitSlop={HITSLOP_10}>
+          hitSlop={HITSLOP_10}
+          style={triggerStyle}>
```
Comment (estrattonbailey) [inline src/view/com/util/forms/NativeDropdown.web.tsx:126]: Needed to pass this down on web

**Review by pfrazee — APPROVED** [review]
Coo coo

**Pull request merged**